bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/config/project/CodeCheckerProject.java

```diff
@@ -147,6 +166,92 @@ private void updateProjectRelated() {
         }
     }
 
+    /**
+     * Merges a build log to an other.
+     * 
+     * @param newLogFile
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused @param tag for 'newLogFile'.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641621)